### DISCUSSION TITLE
No DiscordChatExporter app for Mac?

### DISCUSSION THREAD

**Discussion by Debbb42** [Tyrrrz/DiscordChatExporter#852] [General]
Hi everyone! I'm here because I'm trying to export chat messages from a channel on my Discord server, and doing a Google search brought me here. I followed all the steps (or so I thought) to download; I'm on a Mac, so I selected the Stable Release (DiscordChatExporter.CLI.zip) from the Download section. However, when i open the DiscordChatExporter folder, there is no application there — other than the .exe app that won't open because it's for Windows. I can't figure out where the application is for the Mac (I also downloaded and installed the .NET v6.0 runtime as per the instructions). 

Can anyone advise what I did wrong? Thank you for any help!

**Comment by 96-LB** [comment]
The wiki has a guide [here](https://github.com/Tyrrrz/DiscordChatExporter/wiki/GUI%2C-CLI-and-Formats-explained#using-the-cli).
